hack/components/bump-kubevirt-ipam-controller.sh

```diff
@@ -148,7 +148,7 @@ echo 'Adjust kubevirt-ipam-controller to CNAO'
 
 echo 'Copy manifests'
 shopt -s extglob
-rm -rf data/kubevirt-ipam-controller/!(002-rbac.yaml)
+rm -rf data/kubevirt-ipam-controller/!(002-rbac.yaml|004-primary-udn-networkattachdef.yaml)
```

Comment:
ditto: Let's not reference UDN at the nad, just at the kubevirt CR config on HCO.

Reply:
hmm, I'm not against this, but how will anyone ever understand the connection between the NAD and the binding? shouldn't they have at least something in common? a label maybe?

Reply:
Let's name this 
```
004-primary-udn-kubevirt-binding-networkattachdef.yaml
```

Reply:
maybe just ?
004-primary-udn-kubevirt-binding-nad.yaml